assets/js/script.js

```diff
@@ -1,4 +1,8 @@
-/* Scroll To Top */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-plusplus */
+/* eslint-disable max-len */
```

Comment:
- [ ] Please avoid using ESlint-disable to get around the linter errors. the ESlint errors are usually either demonstrating an underlying issue with your code or showing you that you are not following some coding standards. I'd strongly advice you to just fix the ESlint issues